internal/runner/operation_up.go

```diff
@@ -93,6 +93,10 @@ func (r *Runner) executeUpOp(
 	}
 	releaseResult := result.Release
 
+	if releaseResult.Release.Unimplemented {
+		app.UI.Output("Release not supported for platform, skipping...")
```

Comment:
I think this will happen even if the platform does support releasing but just doesn't have a default releaser AND the user didn't define one.

As such, I'd probably make the message `No release phase specified, skipping...`